### ISSUE TITLE
LSJ file header

### ISSUE THREAD

**Issue by lcerrato** [Eumaeus/cite_lsj_cex#1]
Hi,
Cool to see this!

As part of the CTS conversion work I am systematically reviewing and correcting the file headers for all Perseus works. I have not yet done any work on secondary sources.

The boilerplate headers are typically incorrect, especially in matters of attribution. The LSJ should not include Elli Mylonas or William Merrill.  (It should be David Smith and I only. Bill did not work on Greek and Elli was not part of the project when this work was done). I will be updating the Perseus files accordingly.

There's also a little typo here:
> Lisa Cerrato, William Merrill, Elli Mylonas, and David Smith are the names attached to the origional XML files.

FYI I do not know which version of the file @gcelano has provided, but we have made many subsequent edits to our version of the data and we've branched off from what he has and we did not merge his work into the Perseus text. The most significant problem is that the working Perseus version remains beta code. 
https://github.com/PerseusDL/lexica/wiki

Best,
Lisa

**Comment by jtauber** [comment]
As I myself have worked with PerseusDL/lexica AND Giuseppe's data AND plan to do work with this repo, I'd love to be involved in any efforts to better coordinate synchronisation.

**Comment by lcerrato** [comment]
@jtauber 
Thanks!
Let's connect on this. I am going to do a PR on Giuseppe's data for the immediate header correction/credits and would like a plan for merging back to Perseus (as was the plan once upon a time).

**Comment by gcelano** [comment]
Yes, I did my conversion years ago. It was not trivial, and it was meant to be tested before any merging operation. At this point, I think it can be useful before any other conversion of the most update betacode data.

**Comment by Eumaeus** [comment]
Hi all… thanks for this. I apologize for the long, long, long delay. Lisa, I will get in there and make changes.

**Comment by Eumaeus** [comment]
After doing other things for… good Lord… six years, I am back to engaging with these files. I'm leaving this issue open, and perhaps we can pick up from here.
